docs/INDEX.md

```diff
@@ -0,0 +1,60 @@
+# Interview Assignment - Coop Logistics API
```

Comment:
As discussed, would be great to mentioned whether to implement client or server part, as we don't want to leave a room for different interpretation